modules/apps/message-boards/message-boards-service/src/main/java/com/liferay/message/boards/service/impl/MBMessageLocalServiceImpl.java

```diff
@@ -1470,6 +1474,124 @@ public int getRootDiscussionMessagesCount(
 		return count;
 	}
 
+	@Override
+	public List<MBMessage> getSiteUserMessageBoardMessagesActivity(
+		long groupId, long userId, int start, int end) {
+
+		MBMessageTable aliasMBMessageTable1 = MBMessageTable.INSTANCE.as(
```

Comment:
Rename aliasMBMessageTable1 to mbMessageTable

Reply:
Liferay pattern is "alias" at the beginning. Could be that?